File: sdkwork-app-sdk-csharp/Models/DetectionTypeConfig.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace App.Models
{
    public class DetectionTypeConfig
    {
        public string? Type { get; set; }
        public string? Name { get; set; }
        public bool? Enabled { get; set; }
        public string? Sensitivity { get; set; }
        public double? Threshold { get; set; }
    }
}

File: sdkwork-app-sdk-typescript/src/types/detection-type-config.ts
/** 检测类型配置 */
export interface DetectionTypeConfig {
  /** 检测类型 */
  type?: string;
  /** 类型名称 */
  name?: string;
  /** 是否启用 */
  enabled?: boolean;
  /** 敏感度 */
  sensitivity?: 'low' | 'medium' | 'high';
  /** 阈值 */
  threshold?: number;
}
